code/__DEFINES/atmospherics.dm

```diff
@@ -27,6 +27,23 @@
 /// -14C - Temperature used for kitchen cold room, medical freezer, etc.
 #define COLD_ROOM_TEMP 259.15
 
+/// The minimum heat capacity of a gas
+#define MINIMUM_HEAT_CAPACITY 0.0003
+/// Minimum mole count of a gas
+#define MINIMUM_MOLE_COUNT 0.01
+/// Molar accuracy to round to
+#define MOLAR_ACCURACY  1E-4
+/**
```

Comment:
This doc-comment is going to be applied to MOLES_CELLSTANDARD. Is that intentional?

Reply:
I thought that multiline comments like that weren't documented, i'll move it up over this segment